partner_contact_job_position/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_res_partner_job_position_public","res_partner_job_position group_public","model_res_partner_job_position","base.group_public",1,0,0,0
+"access_res_partner_job_position_portal","res_partner_job_position group_portal","model_res_partner_job_position","base.group_portal",1,0,0,0
+"access_res_partner_job_position_group_partner_manager","res_partner_job_position group_partner_manager","model_res_partner_job_position","base.group_partner_manager",1,1,1,1
```

Comment:
Leave out the 'group_' in the id and name columns in the last two lines, just like it was done on the first two lines, and as is according to conventions.

If you (optionally) want to follow conventions completely, change the name column contents to:
`access.<model name>.<group name>`, so elements separated with dots and group name without group.